ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -186,11 +391,57 @@ class ServicesHomeScreen extends React.Component<Props, State> {
   private onSaveAreasOfInterest = (
     selectedFiscalCodes: Option<Set<string>>
   ) => {
-    const { hideModal } = this.props;
     this.props.dispatchUpdateOrganizationsOfInterestMetadata(
       selectedFiscalCodes
     );
-    hideModal();
+    this.props.hideModal();
+  };
+
+  // This method enable or disable services and update the enableServices props
+  private disableOrEnableAllServices = () => {
+    this.props.disableOrEnableServices(
+      Object.keys(this.props.servicesById),
```

Comment:
Here you have to pass only services relative to the specific tab (local/national/all).
If you do so, user is believing to enable/disable only "national" services for example, while he is actually enable/disable all whole services 
<img src="https://user-images.githubusercontent.com/822471/64978239-5946c080-d8b5-11e9-8d7b-6eed8d9d6475.gif" height="600"/>

Reply:
updated within https://github.com/teamdigitale/io-app/commit/8d83d6e22140e5763cc6f4d3cc81895a94412889